qa/suites/rados/upgrade/jewel-x-singleton/8-workload/rgw-swift.yaml

```diff
@@ -4,7 +4,6 @@ meta:
 tasks:
 - rgw: 
     client.0:
```

Comment:
not 100% sure it's required, but I can't find an example where we don't put a 'null' string when dealing with empty value.